lib/widgets/quickalert_dialog.dart

```diff
@@ -113,8 +116,8 @@ class QuickAlert {
       customAsset: customAsset,
       width: width,
     );
-
-    final child = AlertDialog(
+    
+    Widget child = AlertDialog(
```

Comment:
Hello @ZouheirN 👋🏻 
Thank you for your wonderful contribution.

Try implementing it like this,

Widget child = WillPopScope(
        onWillPop: () => Future.value(disableBackBtn),
        child: AlertDialog(
            contentPadding: EdgeInsets.zero,
            shape: RoundedRectangleBorder(
              borderRadius: BorderRadius.circular(borderRadius),
            ),
        content: QuickAlertContainer(
          options: options,
        ),
      ),
  );
}

Happy Coding 💯

Reply:
Hello, please check my new commit. You are right, and I have implemented it the way you want it (although it should be Future.value(!disableBackBtn)).